Formula/pdftilecut.rb

```diff
@@ -0,0 +1,21 @@
+class Pdftilecut < Formula
+  desc "Sub-divide a PDF page(s) into smaller pages so you can print them"
+  homepage "https://github.com/oxplot/pdftilecut"
+  url "https://github.com/oxplot/pdftilecut/archive/v0.4.tar.gz"
+  sha256 "8a75a0d2e196d156bca4dc006f84c9b7730f20be8c0fc2c90521c538aa627188"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+  depends_on "jpeg-turbo" => :build
+  depends_on "qpdf" => :build
```

Comment:
```suggestion
  depends_on "jpeg-turbo"
  depends_on "qpdf"
```
It seems these are now linked dynamically, so they are no longer build-time dependencies.

Reply:
I definitely don't know what I'm doing. I totally rely on you

Reply:
😄 don't worry, I (kinda) know what I'm doing. At least right now I do